samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -329,18 +321,21 @@ static void generate_modulated_rf_packet(uint8_t mode,
 	tx_packet[0] = sizeof(tx_packet) - 1;
 #endif /* CONFIG_HAS_HW_NRF_RADIO_IEEE802154 */
 
-	/* Fill payload with random data. */
-	for (uint8_t i = 0; i < sizeof(tx_packet) - 1; i++) {
-		if (pattern == TRANSMIT_PATTERN_RANDOM) {
-			tx_packet[i + 1] = rnd8();
-		} else if (pattern == TRANSMIT_PATTERN_11001100) {
-			tx_packet[i + 1] = 0xCC;
-		} else if (pattern == TRANSMIT_PATTERN_11110000) {
-			tx_packet[i + 1] = 0xF0;
-		} else {
-			/* Do nothing. */
-		}
+	switch (pattern) {
+	case (TRANSMIT_PATTERN_RANDOM):
```

Comment:
I think parentheses are redundant (also for the remaining cases)

```suggestion
	case TRANSMIT_PATTERN_RANDOM:
```

Reply:
Yes, fixed.